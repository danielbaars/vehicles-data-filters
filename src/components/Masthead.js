import React from 'react';

import Row from './Row';
import FramedBox from './FramedBox';

const Masthead = () => {
  return (
    <div className='masthead__outer'>
      <Row>
        <div className='small-12 cell'>
          <FramedBox className='masthead'>
            <div className='masthead__logo imgc'>
              <img src={require('../img/traffic-meister-logo.svg')} alt="logo" />
            </div>
            <div className='masthead__visual imgch'>
              <img src={require('../img/vehicles.png')} alt="vehicles" />
            </div>
          </FramedBox>
        </div>
      </Row>
    </div>
  );
};

export default Masthead;
