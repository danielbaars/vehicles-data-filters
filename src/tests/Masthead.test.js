import React from 'react';
import renderer from 'react-test-renderer';

import Masthead from '../components/Masthead';

describe('Masthead', () => {
  const component = <Masthead />;
  it('renders and matches our snapshot', () => {
    const rComponent = renderer.create(component);
    const tree = rComponent.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
