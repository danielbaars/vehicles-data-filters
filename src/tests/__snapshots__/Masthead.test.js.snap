// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Masthead renders and matches our snapshot 1`] = `
<div
  className="masthead__outer"
>
  <div
    className="grid-container"
  >
    <div
      className="grid-x grid-margin-x "
    >
      <div
        className="small-12 cell"
      >
        <div
          className="masthead inner"
        >
          <div
            className="masthead__box box"
          >
            <div
              className="masthead__logo imgc"
            >
              <img
                alt="logo"
                src="traffic-meister-logo.svg"
              />
            </div>
            <div
              className="masthead__visual imgch"
            >
              <img
                alt="vehicles"
                src="vehicles.png"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
